vibe_code_apps_22/eplus_gym_app/dsm_console.py

```
@@ -425,25 +510,72 @@ def stage_idf_for_period(src: Path, dest: Path, begin: str, end: str) -> Path:
+                save_apply_report(
+                    site_root,
+                    {
+                        **report,
+                        "staged_idf": str(dest),
+                        "begin": str(begin)[:10],
+                        "end": str(end)[:10],
+                        "source_idf": str(src),
+                    },
+                )
+            except Exception:  # noqa: BLE001
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)